org.uqbar.project.wollok/src/org/uqbar/project/wollok/scoping/cache/WollokGlobalScopeCache.xtend

```diff
@@ -0,0 +1,11 @@
+package org.uqbar.project.wollok.scoping.cache
+
+import java.util.List
+import org.eclipse.emf.common.util.URI
+import org.eclipse.xtext.xbase.lib.Functions.Function0
+import org.uqbar.project.wollok.wollokDsl.Import
+import org.eclipse.xtext.resource.IEObjectDescription
+
+interface WollokGlobalScopeCache {
```

Comment:
This is to fix the other issue reported by @Juancete about the imports files being loaded many times, right ?